scss/bootstrap/overrides/style/_navs.scss

```diff
@@ -269,3 +199,76 @@
     }
   }
 }
+
+
+//== Nav tabs in a panel-default
+.panel-default {
+
+  // Horizontal nav tabs
+  .nav-tabs:not(.nav-tabs-stacked) {
+    $padding-x: nth($crm-panel-subheading-padding, 2);
+
+    background: $gray-lighter;
+    padding: 5px 0 0 $padding-x;
+  }
+
+  // Vertical nav tabs w/ content on the right, in a grid+flexbox layout
+  .nav-tabs-stacked {
+    border-bottom: 0;
+
+    > li {
+      float: none;
+
+      > a {
+        margin-right: 0;
+        padding: $crm-nav-stacked-link-padding;
+      }
+
+      > a,
+      > a:hover {
+        border-color: transparent $nav-tabs-active-link-hover-border-color transparent transparent;
+      }
+
+      &.active {
+
+        > a,
+        > a:hover,
+        > a:focus {
+          border-color: $nav-tabs-active-link-hover-border-color transparent $nav-tabs-active-link-hover-border-color transparent;
+          border-bottom-color: $nav-tabs-active-link-hover-border-color;
+        }
+      }
+    }
+
+    // The wrapper that enforces the layout
+    &-wrapper {
+      border-top: 1px solid $crm-dark-grayblue;
+      display: flex;
```

Comment:
we officially support IE11 and Edge, so it should be fine. I usually try to avoid using stuff like flexbox if possible just to make sure that even older version are supported (tbh not sure if i'm just being unnecessarily extra-cautious at this point), but if I remember correctly there was no way to achieve this style without Flex